CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+## vNext
```

Comment:
The `vNext` title is an anchor looked for by the release script. The script replaces it with an actual version.